### ISSUE TITLE
Music plugin includes artist names in the track title

### ISSUE THREAD

**Issue by mrBackSlash-it** [lowlighter/metrics#202]
Hello, i have recently noticed that the Music plugin, when put in playlist mode, includes the artists in the track name. I only tested it for Spotify and not for other platforms because i don't have them and i think it's just a parsing (?) problem.

This is from my profile:
![image](https://user-images.githubusercontent.com/58268988/112717578-07411b00-8eee-11eb-8c15-95576c61e105.png)

I think the problem should be somewhere here but i don't know, it's a guess.
https://github.com/lowlighter/metrics/blob/29173250927073f3dda866d203026447c34d571c/source/plugins/music/index.mjs#L97-L106

**Comment by lowlighter** [comment]
Indeed, seems that there were an additional div in-between that got added:
```diff
- name:tr.querySelector("td:nth-child(2) div:nth-child(1)").innerText,
- artist:tr.querySelector("td:nth-child(2) div:nth-child(2)").innerText,
+ name:tr.querySelector("td:nth-child(2) div div:nth-child(1)").innerText,
+ artist:tr.querySelector("td:nth-child(2) div div:nth-child(2)").innerText,
```

This will be fixed in v3.6, thanks for the report 👍🏼

**Comment by mrBackSlash-it** [comment]
No problem! Thanks for the quick reply :D
